foundation_cms/templates/patterns/components/listing_header.html

```diff
@@ -1,7 +1,12 @@
+{% load i18n %}
+
 <div class="listing-header">
     <p>
-        Showing <span id="result-count">{{ items|length }}</span> results for:
+        {% blocktrans %}
+            Showing <span id="result-count">{{ items|length }}</span> results for:
```

Comment:
Hm, I’m not sure this works, I think we have to pass this variable as argument to the blocktrans: https://docs.djangoproject.com/en/5.2/topics/i18n/translation/#blocktranslate-template-tag